dedupe/ckan_api.py

```diff
@@ -181,7 +182,7 @@ def remove_package(self, package_id):
             log.info('Not removing package in dry_run package=%s', package_id)
             return
 
-        self.request('POST', '/action/dataset_purge', json={
+        self.request('POST', '/action/package_delete', json={
```

Comment:
This got changed by my PR, you'll probably need to rebase your changes from main...